packages/components/Button/src/Button.android.tsx

```diff
@@ -54,9 +56,10 @@ export const Button = compose<IButtonType>({
       <Slots.root>
         <Slots.ripple>
           <Slots.stack>
-            {info.icon && <Slots.icon />}
+            {info.startIcon && <Slots.startIcon />}
             {info.content && <Slots.content />}
             {children}
+            {info.endIcon && <Slots.endIcon />}
```

Comment:
quick q do we want to add endIcon into this review?  I noticed it doesn't have an entry in `styles` below

Reply:
I added this for consistency. I'm not sure if the styles are working that's why I didn't add it. Maybe it works for iconFont, I'll check this scenario. Because previously I could change the color of icon only by changing style for the slot.